prprocessor/__main__.py

```diff
@@ -91,6 +118,31 @@ def summarize(summary: Mapping[str, Iterable], show_headers: bool) -> Generator[
                 yield f'* {line}'
 
 
+async def update_pr_labels(pull_request: Mapping, labels_to_add: Iterable[Label],
+                           labels_to_remove: Iterable[Label]) -> None:
+    github_api = RUNTIME_CONTEXT.app_installation_client
+
+    tasks = []
+
+    repository = pull_request['base']['repo']['full_name']
+
+    # TODO: pull_request['labels_url']
+    # https://github.com/orgs/community/discussions/66499
+    url = f'{pull_request["issue_url"]}/labels{{/name}}'
```

Comment:
should this be
```suggestion
    url = f'{pull_request["issue_url"]}/labels/{{name}}'
```

the `/` inside the moustache seems odd

Reply:
No, that's the templating that GitHub uses. https://docs.github.com/en/free-pro-team@latest/rest/issues/issues?apiVersion=2022-11-28#get-an-issue shows:
```
    "labels_url": "https://api.github.com/repos/octocat/Hello-World/issues/1347/labels{/name}",
```

So in theory I could avoid the TODO with retrieving the issue and then getting the labels URL from that.

Reply:
ooooooohkay. /me hides again.